src/WebSdk/Publish/Targets/PublishProfiles/DefaultContainerDeploy.pubxml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Should we just call it DefaultContainer?  The others have Deploy because MSDeploy & ZipDeploy are brand names.